agriculture_shop/__manifest__.py

```diff
@@ -12,7 +12,6 @@
         'pos_sale',
         'product_expiry',
         'purchase_requisition',
-        'payment_demo',
```

Comment:
Could you add a test that ensures payment_demo is not added to the dependency list anymore?